docs/Compatibility.md

```diff
@@ -4,156 +4,247 @@ The Ruby Datadog Trace library is open source. See the [dd-trace-rb][1] GitHub r
+
 [31]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#presto
+
 [33]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#que
+
 [34]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#racecar
+
 [35]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rack
+
 [36]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rails
+
 [37]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rake
+
 [38]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#redis
+
 [39]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#resque
+
 [40]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rest-client
+
 [41]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#roda
+
 [42]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sequel
+
 [43]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#shoryuken
+
 [44]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sidekiq
+
 [45]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sinatra
+
 [46]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sneakers
+
 [47]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#stripe
+
 [48]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sucker-punch
 
+### Support Policy
+
+Datadog APM for Ruby is built upon dependencies defined in specific versions of the host operating system, Ruby
+runtime, certain Ruby libraries, and the Datadog Agent/API. When these versions are no longer supported by their
+maintainers, Datadog APM for Ruby limits its support for these as well.
+
+#### Levels of Support
+
+| **Level**                                              | **Support provided**                                                                                                                                                         |
+|--------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Unsupported</span>      | No implementation. [Contact our customer support team for special requests.](https://www.datadoghq.com/support/)                                                             |
+| <span id="support-beta">Beta</span>                    | Initial implementation. May not yet contain all features. Support for new features, bug & security fixes provided on a best-effort basis.                                    |
+| <span id="support-ga">General Availability (GA)</span> | Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
```

Comment:
To me "GA" sounds like:
- a point in time
- it's feature oriented, not version oriented

e.g in an announcement: "we announce general availability of feature foo" == "feature foo is now generally available".

Inspired by Node, I would like to suggest calling this "current".

Reply:
Now that you mention it, I agree. GA typically indicates a point in time. @delner any objection here?

Reply:
No strong opinion on my end. I will defer. `Current` is fine.

Reply:
I have no objection to either `Current` or `Latest`. I lean slightly towards `Latest` if if provides better consistency. @lloeki you get the deciding vote.

Reply:
Either one is fine to me. `Latest` sounds just as good, let's go for that.